src/integrations/filestorage/implementations/MemoryFS.ts

```diff
@@ -0,0 +1,39 @@
+
+import { FileStorage } from '../definitions/interfaces.js';
+import { FileNotFound } from '../definitions/errors.js';
+
+export default class MemoryFS implements FileStorage
+{
+    #files: Map<string, Buffer> = new Map();
+
+    async connect(): Promise<void> { }
+
+    async disconnect(): Promise<void> { }
```

Comment:
The map should be removed in the disconnect to make sure that we can clear the memory during tests. The same way we are doing for the in memory database.

And I also miss the NotConnected error.